config/config.go

```diff
@@ -427,31 +433,34 @@ func (c *Config) manualString() string {
 	return sb.String()
 }
 
-func (c *Config) merge(additional *Config) error {
-	return mergo.Merge(c, additional, mergo.WithOverride)
+// UseDefault sets the default configuration
+func (b *Builder) UseDefault() *Builder {
+	b.Config = DefaultConfig()
+	return b
 }
```

Comment:
we don't need this method since the `Build` starts with a default. I think we should replace it with `WithConfig` which set the base config.